eve/main.yml

```diff
@@ -291,6 +291,13 @@ stages:
         command: bash scripts/run-e2e-test.sh "end2end" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} ${STAGE} "default"
         workdir: build/eve/workers/end2end
         haltOnFailure: true
+    - ShellCommand:
+        name: Run smoke tests
+        env:
+          <<: [*kind-env, *e2e-env]
+        command: bash scripts/run-e2e-test.sh "smoke" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} ${STAGE} "default"
```

Comment:
```suggestion
        command: bash scripts/run-e2e-test.sh "smoke" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} "smoke" "default"
```

Reply:
Thanks. But why is the previous one ${STAGE} then?
```
    - ShellCommand:
        name: Run init CI test
        env:
          <<: [*kind-env, *e2e-env]
        command: bash scripts/run-e2e-test.sh "end2end" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} ${STAGE} "default"
        workdir: build/eve/workers/end2end
        haltOnFailure: true
```

Reply:
the stage env comes from https://github.com/scality/Zenko/pull/1207/files#diff-5f2489722022b40fce9b08efe051f3bf88356e50c44a6d4ff47376747dd7a006L187

the idea was to allow for different types of test to build of the same stage steps. An example of way of adding the `smoke` test would be:

```
base: &test_stage
   ...

end2end: *base
smoke: *base
```